### DISCUSSION TITLE
Textarea in webapp too short

### DISCUSSION THREAD

**Discussion by fiammanda** [standardnotes/forum#2591] [General] [answered]
![Screenshot 2022-09-03 at 21-08-25 Standard Notes an end-to-end encrypted notes app](https://user-images.githubusercontent.com/8189264/188271867-f554ad1c-1fc3-4630-ac10-79b99adb7344.png)

The following styles will fix.

```
#note-text-editor {
  height: 100%;
}
```

**Comment by onlinemumie** [comment, marked as answer]
This error occurs on [app.standardnotes.org](https://app.standardnotes.org/). But nowdays, the link to the web app is [app.standardnotes.com](https://app.standardnotes.com/).

**Comment by retrocaster** [comment]
I starting intermittently seeing the same issue a few days, and I was also using app.standardnotes.org.
Shouldn't there be an official notification that the site has moved?

**Comment by onlinemumie** [comment]
I think so too. An official notification would have been appreciated.

According to some comments on [the discord channel](https://discord.com/channels/894600255058898974/894600255058898977), the difference between .org and .com doesn't seem to be intensional - they should be the same. Maybe one has to empty the browser's cache or something to get the old page working as before.
